### PR TITLE
update readme as per recent changes

### PR DESCRIPTION
**Pull request by bengl**
### What does this PR do?

Updates the readme.

### Motivation

Lots of changes recently have not been updated in the readme.

### Checklist

[] I have added tests for the code I am submitting
[] My unit tests cover both failure and success scenarios
[] If applicable, I have discussed my architecture

### CODE REVIEW

**Review by brettlangdon — APPROVED** [review]

**Comment by bengl** [comment]
Gonna hold off merging until #22 lands, since it contains changes documented here.

**Pull request merged**
